plugins/LilyPad/LilyPad.cpp

```diff
@@ -1195,19 +1215,27 @@ u8 CALLBACK PADpoll(u8 value) {
 				for (i = 0; i<8; i++) {
 					b2 -= (sum->buttons[i+4] > 0) << i;
 				}
-				if (config.padConfigs[query.port][query.slot].type == GuitarPad && !config.GH2) {
-					sum->sticks[0].horiz = -255;
-					// Not sure about this.  Forces wammy to be from 0 to 0x7F.
-					// if (sum->sticks[2].vert > 0) sum->sticks[2].vert = 0;
+
+				if (config.padConfigs[query.port][query.slot].type == DancePad) {
+					for (i = 4; i<8; i++) {
+						b1 -= (sum->buttons[i + 8]   > 0) << i;
```

Comment:
It was used in the original code to keep the two > 0 of the for loops in the same space. I just copied it without removing the space.